docs/dev/object-deletion.md

```diff
@@ -0,0 +1,118 @@
+# Manifest Annotation For Object Deletion
+
+Developers can remove any of the currently managed CVO objects by modifying an existing manifest and adding the delete annotation .metadata.annotations["release.openshift.io/delete"]="true". This manifest annotation requests the CVO delete the in-cluster object instead of creating/updating it.
```

Comment:
nit: backticks around the code literal: `.metadata.annotations["release.openshift.io/delete"]="true"`

Reply:
fixed